src/routes/categories/components/__tests__/__snapshots__/CategoryListItem.spec.js.snap

```diff
@@ -75,3 +75,95 @@ exports[`CategoryListItem should replace empty thumbnail 1`] = `
   </CategoryListItem__SubCategory>
 </CategoryListItem__Row>
 `;
+
+exports[`CategoryListItem should render and match snapshot 1`] = `
```

Comment:
I think this test is not really helpful. Maybe we should assert on something else than the snapshot.

Reply:
I'm using Firefox